proto/iotics/api/meta.proto

```diff
@@ -117,7 +117,7 @@ message SparqlQueryResponse {
 
     // Result host identifier. Indicates from which host this result chunk came from. For a local result, this field
     // will be unset.
-    HostID remoteHostId = 1;
+    string hostId = 1;
```

Comment:
hostId - is this the right name vs hostTwinDID?

Reply:
Leaving as hostId. Planning to release the hostId endpoint which will initially return peerId.